packages/nuxt/test/pages.test.ts

```diff
@@ -22,6 +24,7 @@ describe('pages:generateRoutesFromFiles', () => {
     description: string
     files?: Array<{ path: string, template?: string, meta?: Record<string, any> }>
     output?: NuxtPage[]
+    focus?: boolean
```

Comment:
Can you explain your thinking behind Focus instead of Flexible Test Variants? I thought it might be good to allow committing failing or todo tests as well.

Reply:
ahhh! I was only imagining a local workflow of skipping them.

Reply:
reverting back!